plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -76,19 +76,23 @@ func init() {
 	// PodFitsPorts has been replaced by PodFitsHostPorts for better user understanding.
 	// For backwards compatibility with 1.0, PodFitsPorts is regitered as well.
 	factory.RegisterFitPredicate("PodFitsPorts", predicates.PodFitsHostPorts)
+	// Fit is defined based on the absence of port conflicts.
+	factory.RegisterFitPredicate("PodFitsHostPorts", predicates.PodFitsHostPorts)
+	// Fit is determined by resource availability.
+	factory.RegisterFitPredicateFactory(
+		"PodFitsResources",
+		func(args factory.PluginFactoryArgs) algorithm.FitPredicate {
+			return predicates.NewResourceFitPredicate(args.NodeInfo)
+		},
+	)
+	// Fit is determined by the presence of the Host parameter and a string match
+	factory.RegisterFitPredicate("HostName", predicates.PodFitsHost)
 }
 
 func defaultPredicates() sets.String {
 	return sets.NewString(
-		// Fit is defined based on the absence of port conflicts.
-		factory.RegisterFitPredicate("PodFitsHostPorts", predicates.PodFitsHostPorts),
-		// Fit is determined by resource availability.
-		factory.RegisterFitPredicateFactory(
-			"PodFitsResources",
-			func(args factory.PluginFactoryArgs) algorithm.FitPredicate {
-				return predicates.NewResourceFitPredicate(args.NodeInfo)
-			},
-		),
+		// PodFitsResources, PodFitsHost and PodFitsHostPorts are moved to GeneralPredicates() which
```

Comment:
Remove this comment. It describes what you did in this PR, but won't be a useful comment to someone reading the code who hasn't seen the old version of the code. I think the comment I suggested for init() will also help.